packages/front-end/pages/features/index.tsx

```diff
@@ -43,6 +44,13 @@ export default function FeaturesPage() {
   const end = start + NUM_PER_PAGE;
 
   const { features, loading, error, mutate } = useFeaturesList();
+  let showToggle = false;
+  for (let i = start; i < end; i++) {
```

Comment:
A couple things:
1. The name `showToggle` is a little vague.  I would call it `showArchiveToggle` or something like that.
2. This is only going to loop through the current page of features.  I think it would be weird if you go to page 2 and all of a sudden a new filter option shows up
3. This loop includes features that have been filtered out by a search term or tag.  That could cause a situation where the toggle button appears broken since it has no effect on the list shown to the user.

To fix the above, I would move this down to line ~78 (after `filtered` is defined) and loop through the entire `filtered` list to check if any features are archived.  And since you'd be checking that whole array, you could also use the [Array.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) method instead of a loop if you want, but that's up to you.

Reply:
Changed in recent commit